### DISCUSSION TITLE
DB issue when deploying on ubuntu server

### DISCUSSION THREAD

**Discussion by okothmax** [firefly-iii/firefly-iii#7750] [Firefly III]
I'm experiencing this problem when deploying on my server, Please assist

![Screenshot (6)](https://github.com/firefly-iii/firefly-iii/assets/112311293/221652b8-eabe-4404-8277-48b2ecffd276)

**Comment by JC5** [comment]
You could Google the error?

**Comment by okothmax** [comment]
I have but no solutions found.

**Comment by JC5** [comment]
Did you install MySQL, and configure Firefly III to connect to it?

**Reply by okothmax** [reply]
I resolve the issue. I configured my mysql to resolve to localhost
